transport/http/outbound.go

```diff
@@ -52,6 +52,14 @@ type outbound struct {
 	URL    string
 }
 
+func (o outbound) Start() error {
+	return nil // nothing to do
```

Comment:
Already mentioned above but: Is this worth the added synchronization cost? You'll need to use either an atomic or a lock when accessing the value from `Call`.